proposals/description/fip_105.ts

```diff
@@ -1,17 +1,133 @@
 import { ProposalDescription } from '@custom-types/types';
 
 const fip_105: ProposalDescription = {
-  title: 'FIP-105: Grant DAI PSM skimmer PCV_CONTROLLER_ROLE',
+  title: 'FIP-105: Reinforce PCV by consolidating assets and perform technical maintenance',
   commands: [
+    /////////  DAI PSM Skimmer   //////////////
     {
       target: 'core',
       values: '0',
       method: 'grantRole(bytes32,address)',
       arguments: ['0x0866eae1216ed05a11636a648003f3f62921eb97ccb05acc30636f62958a8bd6', '{daiFixedPricePSMFeiSkimmer}'],
       description: 'Grant the new DAI PSM Skimmer the PCV_CONTROLLER_ROLE'
+    },
```

Comment:
just flagging that this is unscalable and we should try to revitalize the sink efforts when things settle down

Reply:
As discussed, have removed this in favour of setting an appropriate PCV deposit to be a safe address, pulling funds there and then burning